charts/go-boilerplate-ddd/templates/pdb.yaml

```
@@ -0,0 +1,29 @@
+{{- if and .Values.boilerplate.enabled .Values.boilerplate.pdb.enabled }}
+{{- if and .Values.boilerplate.pdb.minAvailable .Values.boilerplate.pdb.maxUnavailable }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)